examples/tinywl/XdgStackWorkspace.qml

```diff
@@ -167,7 +235,12 @@ Item {
                 surface.outputCoordMapper = surface.CoordMapper.helper.get(output)
             }
 
+            function cancelMinimize () {
+                surfaceConnections.onRequestCancelMinimize();
```

Comment:
Move the inside codes of `onRequestCancelMinimize()` to here, and call `cancelMinimze()` in `onRequestCancelMinimize()`